### DISCUSSION TITLE
nextjs@14.2.3+antd@5.17 next build 直接卡住

### DISCUSSION THREAD

**Discussion by yshinu** [ant-design/ant-design#48803] [General]
我只是在/app/page.tsx里面引入了antd官方的表格实例，dev的时候没问题，打包一直卡在：
![image](https://github.com/ant-design/ant-design/assets/90977980/12cc06d1-82df-4a67-a7e1-714b73ae80e6)

**Comment by li-jia-nan** [comment]
相关 issue：

- #48758
